package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fresh-require",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Sorry, would you mind changing this back? I update the version number with the release. Also, any reason for the 2.0 release, not a 1.1? Otherwise, thanks for the PR!